logger/syslogd/syslogd.go

```diff
@@ -93,6 +93,15 @@ func (h *handler) mainLoop() {
 // Listen starts a new syslog server which runs until it receives a signal.
 func Listen(signalChan chan os.Signal, cleanupDone chan bool) {
 	fmt.Println("Starting syslog...")
+	// If logRoot doesn't exist, create it
+	// equivalent to Python's `if not os.path.exists(filename)`
+	if _, err := os.Stat(logRoot); os.IsNotExist(err) {
+		err = os.Mkdir(logRoot, 0777)
+		if err != nil {
+			fmt.Printf("unable to create logRoot at %s", logRoot)
+			return
```

Comment:
let's change this with a `log.Fatalf` so it exits correctly, otherwise it'll return exit(0)